js/selectors/dashboard-selectors.js

```diff
@@ -98,7 +98,7 @@ export const getStudentAverageProgress = createSelector(
 export const getCurrentActivity = createSelector(
   [ getActivities, getCurrentActivityId ],
   (activities, currentActivityId) => {
-    return activities.find(activity => activity.get("id") === currentActivityId);
+    return activities.find(activity => activity.get("id") === currentActivityId) || activities.first();
```

Comment:
The only problem with this approach is you can't toggle the activity to close it like you can currently on `master`.  I think that is useful to be able to see all the answers.

I think a better approach would be to set currentActivityId at load time to the first activity.  That way it can be toggled to undefined to close it like now.